interfaces/cython/cantera/base.pyx

```diff
@@ -111,8 +109,21 @@ cdef class _SolutionBase:
         elif source:
             root = AnyMapFromYamlString(stringify(source))
 
-        phaseNode = root[stringify("phases")].getMapWhere(stringify("name"),
-                                                          stringify(name))
+        if name == 'gri30_mix' or name == 'gri30_multi':
+            # catch transitional behavior
+            warnings.warn(
+                "The phases 'gri30_mix' and 'gri30_multi' are no longer "
+                "defined in the input file 'gri30.yaml'. The YAML version uses "
+                "the 'Mix' transport manager by default instead, which can be "
+                "overridden by specifying a transport manager via the "
+                "`transport_model' keyword argument during initialization. "
+                "Support for transitional behavior will be removed after "
+                "Cantera 2.5.", DeprecationWarning)
+            phaseNode = root[stringify("phases")].getMapWhere(stringify("name"),
+                                                              stringify("gri30"))
```

Comment:
If I understand the new default correctly, this isn't needed, right?

Reply:
It is still needed as the old names are processed elsewhere.